x-pack/plugins/fleet/common/types/models/settings.ts

```diff
@@ -17,5 +15,3 @@ export interface Settings extends BaseSettings {
   id: string;
   preconfigured_fields?: Array<'fleet_server_hosts'>;
 }
-
-export interface SettingsSOAttributes extends BaseSettings, SavedObjectAttributes {}
```

Comment:
I based this changed on what was done on https://github.com/elastic/kibana/pull/140099. The new type is moved to `so_attributes` below